pkg/controller/plan/kubevirt.go

```diff
@@ -172,34 +174,40 @@ func (r *KubeVirt) EnsureNamespace() (err error) {
 
 //
 // Get the importer pod for a DataVolume.
-func (r *KubeVirt) GetImporterPod(dv DataVolume) (pod *core.Pod, err error) {
+func (r *KubeVirt) GetImporterPod(dv DataVolume) (pod *core.Pod, found bool, err error) {
 	pod = &core.Pod{}
-	name := fmt.Sprintf("importer-%s", dv.Name)
+	if dv.PVC == nil || dv.PVC.Annotations[AnnImporterPodName] == "" {
```

Comment:
This guard on the content of the annotation is to ensure that if it is cleared for any reason (bug in CDI, deliberate change, user action), that we're not calling `Get` with an empty name. I want this to return with found=false and no error rather than pass along the `empty name` error returned from `Get` in that situation.

Reply:
ack, so how about adding a log to indicate that the metadata of the PVC lacks information to differentiate this from a case in which it is set properly and it doesn't match any pod?